javascript_nodejs/2.echobot-with-counter/index.js

```diff
@@ -4,7 +4,7 @@
 const path = require('path');
 const restify = require('restify');
 
-const ERROR = 1;
+const CONFIG_ERROR = 1;
 
 // Import reuqired bot services. See https://ama.ms/bot-services to learn more about the different part of a bot
```

Comment:
spelling. "required"

Reply:
URL is wrong. "ama" != "aka"

Reply:
Fixed.

Reply:
Both the spelling and URL are still wrong. Doesn't look like they were fixed. 

---
In reply to: [214746105](https://github.com/Microsoft/BotBuilder-Samples/pull/305#discussion_r214746105) [](ancestors = 214746105)